src/components/Profile/components/PersonalInfoList/index.tsx

```diff
@@ -271,6 +271,28 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }: Props)
       />
     );
 
+  const getYearFromDate = (dateString: Date | undefined): string => {
+    if (!dateString) return 'Unknown';
+    const date = new Date(dateString);
+    if (date.getFullYear().toString() === '1') return 'Unknown';
+    return date.getFullYear().toString();
+  };
```

Comment:
This pure function should be declared outside of the component, so that it's not recreated on every render.

Also, the types do not match the arguments and their names. You've said that `dateString` is a `Date` (yet named it as a string), and you have to explicitly convert it to a Date. I think it should be `(dateString: string | undefined): string`.